integration/deploy/manifest_test.go

```diff
@@ -487,7 +487,7 @@ func TestDeployOktetoManifestWithinRepository(t *testing.T) {
 
 	require.NoError(t, createOktetoManifest(dir, simpleOktetoManifestContent))
 
-	testNamespace := integration.GetTestNamespace("ifbyol-DeployManifestWithinRepo", user)
+	testNamespace := integration.GetTestNamespace("DeployManifestWithinRepo", user)
```

Comment:
@ifbyol i removed this, was any reason to use your user here?

Reply:
No, I tested it against my dev environment and `ifbyol` prefix was required. I forgot to remove it